docs/features/automation-rules.md

```diff
@@ -244,6 +250,13 @@ This will be translated to Azure Devops integration as:
 ```
 
 In this example a customizable title "Task title" will be added to the Work Item and it will be assigned to "test@test.com". 
+
+Creating a Jira ticket with a parent example:
```

Comment:
Update to "In the following example, the Azure DevOps ticket created from Insights will be associated to an existing parent issue:"

Reply:
done